ci/builders/linux_framework_smoke.json

```diff
@@ -0,0 +1,72 @@
+{
+    "builds": [
+        {
+            "archives": [
+                {
+                    "name": "host_debug_unopt",
+                    "base_path": "out/host_debug_unopt/zip_archives/",
+                    "type": "gcs",
+                    "include_paths": [
+                        "out/host_debug_unopt/zip_archives/dart-sdk-linux-x64.zip"
+                    ],
+                    "realm": "production"
+                }
+            ],
+            "drone_dimensions": [
+                "device_type=none",
+                "os=Linux"
+            ],
+            "gn": [
```

Comment:
As a workaround before the build graph is designed and implemented. The `gn` step can specify a different `--target-dir` to avoid conflicts: https://github.com/flutter/engine/blob/main/tools/gn#L952.